### PR TITLE
Handle claiming rewards multi msg txs

### LINKED ISSUE #3244: Claiming transactions only considers the first claiming tx (of 5 usually)
**Describe the bug**

Claiming transactions in In Activity page only considers the first claiming tx (of 5 usually).

Example tx:

`
https://lcd.nylira.net/txs/DFEAD48410631A06D6EA42F458346EEE71259C02A3F5B33FE6A225E078B131A7`

**To Reproduce**
1. Sign in with address `cosmos18da7h02z3e46ulee5prg5zl6pr9va0ytv6w46k` in explore mode
2. Go to Activity page.

**Expected behavior**

The tx should show all the validators you get your rewards off.  If there are too many,  we can show the message `From x validators`.

**Screenshots**

image

### PR DESCRIPTION
**Pull request by mariopino**
Closes #3244 

**Description:**

Handle claiming rewards multi msg txs

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by mariopino** [comment]
As @faboweb suggested in desktop we should display as many validators as possible. We can shorter long validators names to accomplish that.

**Comment by jbibla** [comment]
very nice.

File: src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
       <h3>{{ caption }}</h3>
       <span>Rewards from&nbsp;</span>
       <router-link
-        :to="`staking/validators/${transaction.value.validator_address}`"
+        :to="`staking/validators/${transaction.value[0].validator_address}`"
```
Comment (colw) [inline src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue]:
```suggestion
        :to="`staking/validators/${validator.validator_address}`"
```

File: src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue [hunk 2]
```diff
@@ -18,9 +18,10 @@
           :alt="`validator logo for ` + validator.name"
         />
         {{
-          transaction.value.validator_address | resolveValidatorName(validators)
+          transaction.value[0].validator_address | resolveValidatorName(validators)
```
Comment (colw) [inline src/components/transactions/message-view/WithdrawDelegationRewardMessageDetails.vue]: Would `validator.name` work here?

**Comment by jbibla** [comment]
go away stalebot. 

@mariopino think you can wrap this up quickly?

**Pull request closed without merging**